redisearch/aggregation.py

```diff
@@ -108,6 +108,49 @@ def build_args(self):
                 ret += ['AS', reducer._alias]
         return ret
 
+class Projection(object):
+    """
+    This object automatically created in the `AggregateRequest.apply()`
+    """
+
+    def __init__(self, alias, projector):
```

Comment:
In 1.6 alias is not mandatory, maybe we can put it as the last argument and give its some default value?